api_definition.go

```diff
@@ -205,8 +205,8 @@ func (a *APIDefinitionLoader) MakeSpec(appConfig *apidef.APIDefinition) *APISpec
 		}
 	}
 
-	newAppSpec.RxPaths = make(map[string][]URLSpec)
-	newAppSpec.WhiteListEnabled = make(map[string]bool)
+	newAppSpec.RxPaths = make(map[string][]URLSpec, len(appConfig.VersionData.Versions))
```

Comment:
Why you specify map length here? From what I understand it may be used only for performance reasons, but I doubt it is the case here.